client/signup/reskinned-processing-screen/index.jsx

```diff
@@ -34,10 +48,14 @@ const useSteps = ( { flowName, hasPaidDomain, hasAppliedDesign } ) => {
 // to work with the onboarding signup flow.
 export default function ReskinnedProcessingScreen( props ) {
 	const { __ } = useI18n();
+	const setupSiteFeatureEnabled = isEnabled( 'signup/setup-site-after-checkout' );
 
 	const steps = useSteps( props );
 	const totalSteps = steps.current.length;
-
+	const isSetupSite =
+		props.flowName === 'setup-site' ||
+		( props.flowName === 'onboarding' && setupSiteFeatureEnabled );
```

Comment:
Unfortunately, there's a bunch of flows that might transition to the `setup-setup` flow. It's a bit of a hack to mention them by name, but maybe it's the best we can do for now.
Ideally, I think this screen would look up what the actual signup destination is to decide whether we're going to the `setup-site` flow, but alas, the destination isn't calculated until the very last minute.

```suggestion
	const flowsThatMightGoToSiteSetup = [ 'business', 'premium', 'personal', 'free', 'onboarding', 'onboarding-with-email', 'onboarding-registrationless', 'business-monthly', 'premium-monthly', 'personal-monthly', 'do-it-for-me' ];
	const isSetupSite =
		props.flowName === 'setup-site' ||
		( flowsThatMightGoToSiteSetup.includes( props.flowName ) && setupSiteFeatureEnabled );
```

Reply:
Oh I think we do actually have access to the ultimate destination, and in fact it's being accessed already in the parent component, and used to determine the `hasDomainApplied` prop! I've updated this PR to pass in a flag to represent this. So now, using this flag, we will not need to test the feature flags or flow names directly; as they will have already been checked :)

Reply:
Sweet!